app/src/main/java/org/thoughtcrime/securesms/database/SmsDatabase.java

```diff
@@ -105,7 +105,7 @@ public class SmsDatabase extends MessagingDatabase {
       PROTOCOL, READ, STATUS, TYPE,
       REPLY_PATH_PRESENT, SUBJECT, BODY, SERVICE_CENTER, DELIVERY_RECEIPT_COUNT,
       MISMATCHED_IDENTITIES, SUBSCRIPTION_ID, EXPIRES_IN, EXPIRE_STARTED,
-      NOTIFIED, READ_RECEIPT_COUNT, UNIDENTIFIED,
+      NOTIFIED, READ_RECEIPT_COUNT, UNIDENTIFIED, HAS_MENTION,
```

Comment:
I'll just add the note here so you see it in the PR: there may be another reference that needs to be updated at line 294

Reply:
I can't seem to find any other areas in `SmsDatabase` or `MmsDatabase` which need it, there were a couple of places in `MmsSmsDatabase` which needed it but I'd already added it to those ones

Reply:
ah yep it was `MmsSmsDatabase` I was thinking of :joy: